runtime/parachains/src/paras_inherent.rs

```diff
@@ -174,24 +174,27 @@ pub mod pallet {
 			// Handle disputes logic.
 			let current_session = <shared::Pallet<T>>::session_index();
 			let freed_disputed: Vec<(_, FreedReason)> = {
-				let fresh_disputes = T::DisputesHandler::provide_multi_dispute_data(disputes)?;
+				let new_current_dispute_sets: Vec<_> = disputes
+					.iter()
+					.filter(|s| s.session == current_session)
+					.map(|s| (s.session, s.candidate_hash))
+					.collect();
+
+				T::DisputesHandler::provide_multi_dispute_data(disputes)?;
```

Comment:
1. previously this was called on all disputes and now only on current session disputes,
2. we're ignoring the return values and this is the only call site it seems, suggesting a refactoring